clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -5082,8 +5180,13 @@ void CGDebugInfo::EmitFunctionEnd(CGBuilderTy &Builder, llvm::Function *Fn) {
   }
   FnBeginRegionCount.pop_back();
 
-  if (Fn && Fn->getSubprogram())
+  if (Fn && Fn->getSubprogram()) {
+    // For each call instruction emitted, add the call site target metadata.
```

Comment:
nit: comment mentions call instruction, but we pass nullptr for the call here - can the comment be re-wording?

Reply:
Comment reworded:
```
    // As the debug info for the given function has been emitted, add its
    // name and node to the call site target information.
```